sdk/typescript/src/graphql/client.ts

```diff
@@ -884,8 +890,77 @@ export class GraphQLSuiClient extends SuiClient {
 	override async devInspectTransactionBlock(
 		input: DevInspectTransactionBlockParams,
 	): Promise<DevInspectResults> {
-		void input;
-		throw new Error('Method not implemented.');
+		// TODO handle epoch
```

Comment:
Note that the epoch parameter was removed from the GraphQL API because it was ignored by the fullnode API!